frontend/src/metabase/redux/embed.js

```diff
@@ -5,10 +5,11 @@ import {
 } from "metabase/lib/redux";
 
 const DEFAULT_OPTIONS = {
-  top_nav: false,
+  top_nav: true,
```

Comment:
Changing according to [the new spec](https://www.notion.so/metabase/Improve-collection-breadcrumbs-with-embedding-parameters-51dd704168e344158b649415fb693960#0ef917ff09244d549fa3a269524c8378)